### DISCUSSION TITLE
Is there a way to contribute to docs?

### DISCUSSION THREAD

**Discussion by MisterSammy** [filamentphp/filament#8346] [General]
Hey, I spotted a tiny typo on https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme.

(`admim\` should be `admin\`). 

I poked around to see if I could make a PR for this, but it wasn't immediately apparent if that was possible.

**Comment by leandrocfe** [comment]
Yes, please
https://github.com/filamentphp/filament/edit/3.x/packages/panels/docs/11-themes.md

**Comment by danharrin** [comment]
There is also an edit button on each page

**Comment by MisterSammy** [comment]
Thanks both!
